templates/vue-demo-store/components/listing-filters/ListingFilters.vue

```diff
@@ -133,6 +133,19 @@ const selectedOptionIds = computed(() => [
   ...searchSelectedFilters.properties,
   ...searchSelectedFilters.manufacturer,
 ]);
+const showResetFiltersButton = computed<boolean>(() => {
+  if (
+    selectedOptionIds.value.length !== 0 ||
+    searchSelectedFilters["max-price"] ||
+    searchSelectedFilters["min-price"] ||
+    searchSelectedFilters["rating"] ||
+    searchSelectedFilters["shipping-free"]
```

Comment:
We should create more flexible functions, there can be custom filters.
Something like this: 
```suggestion
    searchSelectedFilters.some(element => {
     return element !== '' && !!element
    })
```
What do you think?

Reply:
`searchSelectedFilters` has `reactive` type, it's not iterable. But for future would be worth refactoring that 👌